recipes/carna/meta.yaml

```diff
@@ -0,0 +1,34 @@
+package:
+  name: carna
+  version: "1.3.1"
+
+about:
+  home: http://www.bioinf.uni-freiburg.de/Software/Carna/
+  license: GPL
+  license_file: COPYING
+  summary: Multiple alignment of RNAs
+
+build:
+  number: 0
+  skip: True # [osx]
+
+source:
+  fn: carna-1.3.1.tar.gz
+  url: "http://www.bioinf.uni-leipzig.de/~will/Software/CARNA/carna-1.3.1.tar.gz"
+  sha256: 7c78463542c48322e2770eff544b46cffe5e4ce67f5fc58c05b4ead6063b541a
+
+requirements:
+  build:
+    - gecode >=4.4.0
+    - viennarna >=2.2.5
+    - locarna >=1.8.9
+    - gcc # [linux]
+    - libgcc # [linux]
```

Comment:
Hi @bgruening, here are some recipes with (supposedly unnecessary)
libgcc build dependency:

bowtie
fastx_toolkit
gnu-getopt
gnuplot
intarna
last
libgd
libgtextutils
locarna
segemehl

at least it's not only in 'my' recipes ;)

On 04/27/2016 01:30 PM, Björn Grüning wrote:
> In recipes/carna/meta.yaml
> <https://github.com/bioconda/bioconda-recipes/pull/1398#discussion_r61242182>:
> 
>> +build:
>> +  number: 0
>> +  skip: True # [osx]
>> +
>> +source:
>> +  fn: carna-1.3.1.tar.gz
>> +  url: "http://www.bioinf.uni-leipzig.de/~will/Software/CARNA/carna-1.3.1.tar.gz"
>> +  sha256: 7c78463542c48322e2770eff544b46cffe5e4ce67f5fc58c05b4ead6063b541a
>> +
>> +requirements:
>> +  build:
>> +    - gecode >=4.4.0
>> +    - viennarna >=2.2.5
>> +    - locarna >=1.8.9
>> +    - gcc # [linux]
>> +    - libgcc # [linux]
> 
> Feel free to drop me a list of those and I will see if it needs to be
> fixed. On Apr 27, 2016 12:59 PM, Sebastian Will
> <notifications@github.com> wrote:In recipes/carna/meta.yaml:
> You should be right about the libgcc build dependency. Although, I have
> seen several other recipes with the same redundancy. —You are receiving
> this because you were mentioned.Reply to this email directly or view it
> on GitHub
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/bioconda/bioconda-recipes/pull/1398/files/3a95113d47c0c192dd665b8e69dd12773a10edcd#r61242182>
> 


-- 
Dr. Sebastian Will
Bioinformatics/IZBI
Härtelstrasse 16-18, Room 448.1
University Leipzig
D-04107 Leipzig

Tel.: +49 (0) 341 97 16689